src/libraries/System.Net.Http/tests/UnitTests/Headers/NameValueHeaderValueTest.cs

```diff
@@ -16,7 +16,7 @@ public class NameValueHeaderValueTest
         [Fact]
         public void Ctor_NameNull_Throw()
         {
-            AssertExtensions.Throws<ArgumentException>("name", () => { NameValueHeaderValue nameValue = new NameValueHeaderValue(null); });
+            AssertExtensions.Throws<ArgumentException>("name", () => { NameValueHeaderValue nameValue = new NameValueHeaderValue(null as NameValueHeaderValue); });
```

Comment:
It used to test the `string` constructor, not the one you made internal ...
```suggestion
            AssertExtensions.Throws<ArgumentException>("name", () => { NameValueHeaderValue nameValue = new NameValueHeaderValue(null as string); });
```